bf2fc6cc711aee1a0c2a/kas-fleet-manager
# ci: add a check to verify that make generate has been run

## Description


Add a check to verify that make generate has been run
This is only added for Github CI for now. No need to add it to midstream pipelines

PS: I'll add another commit that introduce the required changes with `make generate code/fix` output.
Opening it to get the feedback from CI if it is working. 
 
/cc @miguelsorianod what do you think?
## Verification Steps


1. See the failure reason in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/runs/6472293220?check_suite_focus=true which is an output we get when `make generate code/fix` has not been run.
2. If the `make generate code/fix` command has been run, the CI should pass as normal

## Checklist (Definition of Done)
